packages/scheduled-tasks/src/lib/types/ScheduledTasksTaskOptions.ts

```diff
@@ -0,0 +1,7 @@
+export type ScheduledTasksTaskOptions = {
```

Comment:
So does this work now or not?

Reply:
This one does yes. But this is a different one than the piece's options.

Reply:
Ok